Skuid_Techniques/Multi_Part_Form/README.md

```diff
@@ -0,0 +1,33 @@
+# Multi-part Form Navigation and Real-time Completion Status
+
+Skuid can be used to create very effective complex data-entry forms.  There are two principles that can help users work through an extensive data-entry process:  
```

Comment:
```suggestion
Skuid can be used to create very effective and complex data entry forms.  There are two principles that can help users work through an extensive data-entry process:  
```

A suggestion I'm making throughout: the more common form of this type of flow is called `data entry form` as opposed to the hyphenated `data-entry form`. My brain had to do a double take, but [the Ngram does not lie](https://books.google.com/ngrams/graph?content=data-entry+form%2C+data+entry+form&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cdata%20-%20entry%20form%3B%2Cc0%3B.t1%3B%2Cdata%20entry%20form%3B%2Cc0#t1%3B%2Cdata%20-%20entry%20form%3B%2Cc0%3B.t1%3B%2Cdata%20entry%20form%3B%2Cc0). For the sake of discoverability, let's go with the more common form.